niworkflows/anat/mni.py

```diff
@@ -164,35 +170,115 @@ def _config_ants(self, ants_settings):
 
     def _generate_report(self):
         """ Generates the visual report """
-        NIWORKFLOWS_LOG.debug('Generating visual report')
+        NIWORKFLOWS_LOG.info('Generating visual report')
 
-        def _plot_xyz(anat_img, plot_params, div_id):
+        def _plot_xyz(anat_img, div_id, plot_params=None,
+                      order=('z', 'x', 'y'), cuts=None):
             """
-
+            Plots the foreground and background views
+            Default order is: axial, coronal, sagittal
             """
-            NIWORKFLOWS_LOG.info('Plotting %s', str(anat_img))
-            svgs = []
+            if plot_params is None:
+                plot_params = {}
 
-            for mode, cuts in list(DEFAULT_CUTS.items()):
+            # Use default MNI cuts if none defined
+            if cuts is None:
+                cuts = DEFAULT_MNI_CUTS.copy()
+
+            NIWORKFLOWS_LOG.debug('Plotting %s for interface report.', anat_img)
+            out_files = []
+
+            # Plot each cut axis
+            for mode in list(order):
+                out_file = '{}_{}.svg'.format(div_id, mode)
                 plot_params['display_mode'] = mode
-                plot_params['cut_coords'] = cuts
-                image = plot_anat(anat_img, **plot_params)
-                svgs.append('<div id="{0}-{1}" class="{0}">{2}</div>'.format(
-                    div_id, mode, report.as_svg(image)))
-                image.close()
-            return '\n'.join(svgs)
-
-        plot_params = {'cut_coords': 3}
-        report.save_html(
```

Comment:
Currently, save_html provides validation so we know that the html is valid for its currently-planned use-case: the final report aggregator. As I commented below, I'll make the validation stand-alone from `save_html` so you can use it here.

BTW human-readable reports should have some semantic information about what the image is based on, the title, the original files, etc. Are those included in the SVG? I don't see it but I'm not familiar with SVGs or the `lxml` module so I could be wrong.